src/sector_product.jl

```diff
@@ -102,18 +98,18 @@ end
 ×(c1::NamedTuple, c2::AbstractSector) = ×(SectorProduct(c1), SectorProduct(c2))
 ×(c1::AbstractSector, c2::NamedTuple) = ×(SectorProduct(c1), SectorProduct(c2))
 
-function ×(l1::LabelledInteger, l2::LabelledInteger)
-  c3 = label(l1) × label(l2)
-  m3 = unlabel(l1) * unlabel(l2)
-  return labelled(m3, c3)
+function ×(sr1::SectorUnitRange, sr2::SectorUnitRange)
+  isdual(sr1) == isdual(sr2) || throw(ArgumentError("SectorProduct duality must match"))
+  return sectorrange(
+    sector(sr1) × sector(sr2),
+    sector_multiplicity(sr1) * sector_multiplicity(sr2),
+    isdual(sr1),
+  )
```

Comment:
This seems to ignore any offsets in `sr1` and `sr2`, maybe it should be restricted to `SectorOneTo`.